packages/edit-site/src/components/error-boundary/warning.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Warning } from '@wordpress/block-editor';
+import { useCopyToClipboard } from '@wordpress/compose';
+
+function CopyButton( { text, children } ) {
+	const ref = useCopyToClipboard( text );
+	return (
+		<Button variant="secondary" ref={ ref }>
+			{ children }
+		</Button>
+	);
+}
+
+export default function ErrorBoundaryWarning( {
+	message,
+	error,
+	reboot,
+	dashboardLink,
+} ) {
+	const actions = [];
+
+	if ( reboot ) {
+		actions.push(
+			<Button key="recovery" onClick={ reboot } variant="secondary">
+				{ __( 'Attempt Recovery' ) }
+			</Button>
+		);
+	}
+
+	if ( error ) {
+		actions.push(
+			<CopyButton key="copy-error" text={ error.stack }>
```

Comment:
I think we need to add a fallback here. When I add an empty `front-page.php` to Twenty Twenty-two and open the Site Editor I see an error ("The editor is unable to find a block template for the homepage.") but pressing _Copy Error_ does nothing.

In React DevTools I can see that there is no `text`.

<img width="277" alt="Screen Shot 2022-02-15 at 12 04 42" src="https://user-images.githubusercontent.com/612155/153972947-7c4db5d6-3b90-42c5-80fe-d507784cc1da.png">

Reply:
So somewhere in the code isn't passing a proper error object? I can do some debugging, but if you have any clues let me know.

Reply:
No clues, sorry. It was pretty easy to reproduce, though. Add an empty `front-page.php` file to the top level of `twentytwentytwo`.